app/src/main/java/com/globant/calculatoronboardingkotlin/mvp/contracts/CalculatorContracts.kt

```diff
@@ -1,19 +1,28 @@
 package com.globant.calculatoronboardingkotlin.mvp.contracts
 
-import com.globant.calculatoronboardingkotlin.mvp.presenter.ErrorMessages
-
 interface CalculatorContracts {
     interface Model {
         var firstNumber: String
```

Comment:
Maybe you can move this attributes on the Model class

Reply:
removed, thanks